### ISSUE TITLE
Browser productivity enhancement: add option to have <enter> move cursor to next row and enter edit cell mode

### ISSUE THREAD

**Issue by audiomuze** [pawelsalawa/sqlitestudio#3566]
### Details
Presently pressing enter after completing an edit leaves the cursor positioned on the current cell and exist edit mode.

It'd be a boost to utility/productivity to have the option to have the cursor position itself and enter edit mode on the corresponding cell of the next record when enter is pressed.

**Comment by pawelsalawa** [comment]
I wonder about entering edition mode. I mean if we move cell selection to the next cell, then you don't have to enter edition mode, because if you start typing, it will enter edition mode automatically then.

**Comment by audiomuze** [comment]
Yes, but then you'll be overwriting the existing field value rather than editing it.

**Comment by pawelsalawa** [comment]
Oh, I see. Thanks!
